kubeflow/common/asm/Makefile

```diff
@@ -3,7 +3,13 @@
 # For example: 1.9.2-asm.1+config4:install_asm_1.9.2-asm.1-config4
 # The part before colon symbol should be used for ASM_PACKAGE_VERSION.
 # The part after colon symbol should be used for INSTALL_ASM_SCRIPT_VERSION.
-# You might need to change corresponding kpt value in kubeflow/kpt-set.sh if you upgrade 
+# IMPORTANT: You must change corresponding kpt value in kubeflow/env.sh if you upgrade 
+# For example: export ASM_LABEL=asm-193-2
+# Then go back to kubeflow/ directory, run the following commands:
+# 1. source ./env.sh
+# 2. bash ./kpt-set.sh
+# 3. make apply
+# The above commands allow you to set new namespace labels and re-apply your resources.
```

Comment:
This part seems problematic to me, if users need to upgrade asm package, they should be able to do so directly here. Otherwise, they can only apply Kubeflow as a whole, but not parts.

You might need to learn how to set up a sub kpt package in this directory, in that way, you can add kpt setters in Kptfile of this package. hint: refer to https://googlecontainertools.github.io/kpt/reference/pkg/init/

Reply:
Thank you for the reference! 

Migrate ASM to a newer version in cluster requires re-applying workload: https://cloud.google.com/service-mesh/docs/scripted-install/gke-upgrade#deploying_and_redeploying_workloads. So we need to restart workload. Let's discuss more about how to do it for other Kubeflow components.

But I think it makes sense in terms of setting a kpt package to read the ASM label value.

Reply:
I have updated the ASM upgrade process and local environment variable + kpt config,